rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+- [Example use in the Scaled Agile Framework](https://www.scaledagileframework.com/spikes/)
+
+**Milestones** represent a four week timeline/duration in a Release.
+
+**Releases** are composed of three milestones and occur on a quarterly cadence.
+
+The **Product Backlog** is the set of User Stories and Spikes scheduled for the next Release based on Roadmap priorities. The **Engineering Plan** is another term for the Product Backlog, and is used interchangeably.
+
+## Detailed Design for Roadmaps
+
+The Product Roadmap creates a shared understanding across the DCP community and Users of the DCP for investments and direction for the next year. The Roadmap process MUST occur on an annual basis. 
+
+Themes and Objectives from the Roadmap are the primary source of priorities for planning Releases each quarter.
+
+### Timebox for Roadmaps
+
+All timeboxes are noted as business days. Business days are included regardless of national holidays.
+
+Overlapping timeboxes denote concurrent work.
+
+Time boxes with a range of days indicate that the work MAY be completed at any point during the timebox, but MUST be completed no later than the final day of the timebox.
+
+**Note**: *Input from the Technical Architecture team is integrated on a quarterly basis during Release planning.*
+
+Day 1 in the Roadmap timeline begins in the third quarter.
+
+* **Day 1 - Day 25: [Gathering and reviewing input](#gathering-and-reviewing-input)**
+  * Prerequisite: Science Roadmap
+  * Day 1-25: DCP User Survey 
+  * Day 1-10: Roadmap Retrospective
+* **Day 1 - Day 40: [Drafting the roadmap](#drafting-the-roadmap)**
+  * Day 1-10: Identify a Roadmap Shepherd
+  * Day 25-30: Create themes
+  * Day 31: Collate themes
+  * Day 32-40: Refine themes
+* **Day 40 - Day 55: Receiving feedback**
+  * Day 40: Publish draft Roadmap
+  * Day 41-50: Community Review period 
+  * Day 51-55: Complete final edits
+* **Day 55 - Day 65 : [Iterating with Oversight](#iterating-with-oversight)**
+  * Day 55: Send Roadmap to Oversight
+  * Day 56-65: Review and edit roadmap with Oversight
+  * Day 65: Receive Roadmap approval from Oversight
+  * Day 65: Communicate approved Roadmap 
+
+### Gathering and Reviewing Input
+[Gathering and Reviewing Input]:#gathering-and-reviewing-input
+
+#### Science Roadmap
+
+Oversight MUST provide an annual Science Roadmap to Project Leads to be reviewed at the next Oversight meeting following its receipt.
+
+Project Leads MUST clarify outstanding questions about the Science Roadmap with Oversight within two weeks.
+
+#### Technical Architecture Objectives 
+
+Through a process of their own choosing, the Technical Architecture team MAY document critical architectural issues and technical debt in a Technical Objectives Informational RFC for consideration during each Release. The RFC SHOULD outline technical strategies and estimate cost (such as two engineers at 20%) to realize its objectives. 
```

Comment:
MAY is not good enough. Has to be MUST otherwise it will not be prioritized among the sea of other work. And I strongly think this needs to occur even if tech-arch decides there are no critical issues/tech debt that should be addressed in that release. This checkpoint also needs to be explicitly in the timeline.

Reply:
Please see the related [conversation](https://github.com/HumanCellAtlas/dcp-community/pull/92#discussion_r307984887) with Arathi. I appreciate your strong statement of support for her position.

Reply:
Addressed by https://github.com/HumanCellAtlas/dcp-community/pull/92/commits/4bb18c65af5b244460cf404465b1dbc4ee0cc65e